sites/_index.en.md

```diff
@@ -8,7 +8,7 @@ tags = ["http", "site"]
 
 # Websites
 
-To manage your sites, add the addresses in **Web > Sites**. Choose the [language]({{< ref "languages" >}}) or type of you choice and configure them.
+To access to an address hosted at alwaysdata in HTTP/HTTPS it **must** be declared in **Web > Sites**. Choose the [language]({{< ref "languages" >}}) or type of you choice and configure them.
```

Comment:
Mauvaise traduction : To access to an address hosted at alwaysdata in HTTP/HTTPS.